### PR TITLE
fix: added ConsultingTypeService and refactored the ConsultingTypeSet…

### PR DESCRIPTION
**Pull request by Schweizer-Philipp**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -98,6 +102,11 @@ cache.agencies.configuration.eternal=false
 cache.agencies.configuration.timeToIdleSeconds=0
 cache.agencies.configuration.timeToLiveSeconds=10800
 
+cache.consulting.type.configuration.maxEntriesLocalHeap=100
+cache.consulting.type.configuration.eternal=false
+cache.consulting.type.configuration.timeToIdleSeconds=0
+cache.consulting.type.configuration.timeToLiveSeconds=10800
```
Comment (daho4b) [inline src/main/resources/application.properties:108]: I would suggest increasing the value for 'timeToLiveSeconds'

File: src/test/java/de/caritas/cob/userservice/api/service/ConsultingTypeServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  private ConsultingTypeService consultingTypeService;
+
+  @Mock
+  private ConsultingTypeControllerApi consultingTypeControllerApi;
+
+  @Mock
+  private SecurityHeaderSupplier securityHeaderSupplier;
+
+  @Test
+  public void setup() {
```
Comment (daho4b) [inline src/test/java/de/caritas/cob/userservice/api/service/ConsultingTypeServiceTest.java:33]: Please name the test method speaking

**Review by daho4b — CHANGES_REQUESTED** [review]

File: src/main/java/de/caritas/cob/userservice/api/admin/service/ConsultingTypeAdminService.java [hunk 3]
```diff
@@ -45,28 +48,31 @@ public ConsultingTypeAdminResultDTO findConsultingTypes(Integer page, Integer pe
...
-    return consultingTypeManager.getConsultingTypeSettingsMap().values().stream().sorted(
-        Comparator.comparing(ConsultingTypeSettings::getConsultingTypeUrlName))
-        .map(this::fromConsultingTypeSettings)
-        .collect(Collectors.toList());
+    return null;
+    /*consultingTypeManager.getConsultingTypeSettingsMap().values().stream().sorted(
+    Comparator.comparing(ExtendedConsultingTypeResponseDTO::getConsultingTypeUrlName))
+    .map(this::fromConsultingTypeSettings)
+    .collect(Collectors.toList());*/
+    //TODO
```
[the remark this answers is not in the document]
Comment (Schweizer-Philipp) [reply src/main/java/de/caritas/cob/userservice/api/admin/service/ConsultingTypeAdminService.java:56]: This class was removed in Feat/dob 61 remove consultingtype endpoint from admin api

File: src/main/java/de/caritas/cob/userservice/api/service/AskerImportService.java [hunk 4]
```diff
@@ -578,7 +575,7 @@ private ImportRecordAsker getImportRecordAsker(CSVRecord record) {
     importRecord.setUsernameEncoded(userHelper.encodeUsername(StringUtils.trim(record.get(1))));
     String email = StringUtils.deleteWhitespace(
         record.get(2).trim().equals(StringUtils.EMPTY) ? "" : record.get(2).trim());
-    if (email != null && !email.equals(StringUtils.EMPTY)
```
[the remark this answers is not in the document]
Comment (Schweizer-Philipp) [reply src/main/java/de/caritas/cob/userservice/api/service/AskerImportService.java:581]: email can't be null there, the IDE has noted that

File: src/main/java/de/caritas/cob/userservice/api/service/AskerImportService.java [hunk 5]
```diff
@@ -607,7 +604,7 @@ private ImportRecordAskerWithoutSession getImportRecordAskerWithoutSession(CSVRe
...
```
[the remark this answers is not in the document]
Comment (Schweizer-Philipp) [reply src/main/java/de/caritas/cob/userservice/api/service/AskerImportService.java:610]: email can't be null there, the IDE has noted that

File: src/test/java/de/caritas/cob/userservice/api/admin/service/ConsultingTypeAdminServiceIT.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package de.caritas.cob.userservice.api.admin.service;
+/*package de.caritas.cob.userservice.api.admin.service;
```
[the remark this answers is not in the document]
Comment (Schweizer-Philipp) [reply src/test/java/de/caritas/cob/userservice/api/admin/service/ConsultingTypeAdminServiceIT.java:1]: This class was removed in Feat/dob 61 remove consultingtype endpoint from admin api

**Review by PepeCortisona — APPROVED** [review]

**Pull request closed without merging**